layouts/plugins/system/webauthn/manage.php

```diff
@@ -0,0 +1,127 @@
+ *
+ * @var   FileLayout $this        The Joomla layout renderer
+ * @var   array      $displayData The data in array format. DO NOT USE.
+ *
+ * Layout specific data
+ *
+ * @var   User       $user        The Joomla user whose passwordless login we are managing
+ * @var   bool       $allow_add   Are we allowed to add passwordless login methods
+ * @var   array      $credentials The already stored credentials for the user
+ * @var   string     $error       Any error messages
+ */
+
+/**
+ * Note about the use of short echo tags.
+ *
+ * Starting with PHP 5.4.0, short echo tags are always recognized and parsed regardless of the short_open_tag setting
+ * in your php.ini. Since we only support *much* newer versions of PHP we can use this construct instead of regular
+ * echos to keep the code easier to read.
+ */
+
+// Extract the data. Do not remove until the unset() line.
+extract(array_merge([
+	'user'        => Factory::getApplication()->getIdentity(),
+	'allow_add'   => false,
+	'credentials' => [],
+	'error'       => '',
+], $displayData));
+
+HTMLHelper::_('stylesheet', 'plg_system_webauthn/backend.css', [
+	'relative' => true,
+]);
+
+/**
+ * Why not push these configuration variables directly to JavaScript?
+ *
+ * We need to reload them every time we return from an attempt to authorize an authenticator. Whenever that
+ * happens we push raw HTML to the page. However, any SCRIPT tags in that HTML do not get parsed, i.e. they
+ * do not replace existing values. This causes any retries to fail. By using a data storage object we circumvent
+ * that problem.
+ */
+$randomId    = 'plg_system_webauthn_' . UserHelper::genRandomPassword(32);
+$publicKey   = $allow_add ? base64_encode(CredentialsCreation::createPublicKey($user)) : '{}';
+$postbackURL = base64_encode(rtrim(Uri::base(), '/') . '/index.php?' . Joomla::getToken() . '=1');
+?>
+<div class="plg_system_webauthn" id="plg_system_webauthn-management-interface">
+    <span id="<?= $randomId ?>"
+          data-public_key="<?= $publicKey ?>"
+          data-postback_url="<?= $postbackURL ?>"
+    ></span>
+
+	<?php if (is_string($error) && !empty($error)): ?>
+        <div class="alert alert-danger">
+			<?= htmlentities($error) ?>
+        </div>
+	<?php endif; ?>
+
+    <table class="table table-striped">
+        <thead class="thead-dark">
+        <tr>
+            <th><?= Text::_('PLG_SYSTEM_WEBAUTHN_MANAGE_FIELD_KEYLABEL_LABEL') ?></th>
+            <th><?= Text::_('PLG_SYSTEM_WEBAUTHN_MANAGE_HEADER_ACTIONS_LABEL') ?></th>
+        </tr>
+        </thead>
+        <tbody>
+		<?php foreach ($credentials as $method): ?>
+            <tr data-credential_id="<?= $method['id'] ?>">
+                <td><?= htmlentities($method['label']) ?></td>
+                <td>
+                    <button onclick="return plgSystemWebauthnEditLabel(this, '<?= $randomId ?>');"
+                       class="btn btn-secondary">
+                        <span class="icon-edit icon-white"></span>
```

Comment:
Please update for accessibility all icons following this example

```suggestion
                        <span class="icon-edit icon-white" aria-hidden="true"></span>
```

Reply:
Thank you, I am applying this to all icons now.